libs/nestjs-admin/src/adminSite.ts

```diff
@@ -62,7 +62,7 @@ class DefaultAdminSite {
     // @debt architecture "williamd: this should probably be moved to a Form class"
     // Also, this whole function is a mess of state and special cases. We need to find
     // a better way to organise this.
-    const propertyNames = Object.keys(values)
+    const propertyNames = metadata.columns.map(column => column.propertyName)
```

Comment:
Mh, don't think this is a good idea to rely on the columns rather than what was sent. It probably breaks the manytomany and will make it harder to later handle custom fields, for example. Why do you need to do that?